pytest_testdox/plugin.py

```diff
@@ -14,16 +15,30 @@ def pytest_addoption(parser):
         '--testdox', action='store_true', dest='testdox', default=False,
         help='Report test progress in testdox format'
     )
+    group.addoption(
+        '--force-testdox', action="store_true",
+        dest="force_testdox", default=False,
+        help=(
+            "Force testdox output even when not in real terminal"
+        )
+    )
```

Comment:
What about using a configuration flag, with `parser.addini`? Having either `testdox` and `force-testdox` as command-line arguments may be a bit confusing?

Reply:
One more thing: If you accept the suggestion of using an `ini` setting, what about naming it just `testdox`? I remembered that at some point in the past (#17) I wanted to have it (for different reasons), but decided to close the issue because there weren't strong reasons for that (until now).

Reply:
@renanivo do you think better to remove `--testdox` in favor of only `testdox` in `ini` file?

Reply:
Never mind. I meant that the behavior of `force_testdox` should be what the `textdox` flag in `.ini` file does, but as I wrote it felt much more confusing.